templates/_layout.html

```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="{% block meta_description %}Snaps are containerised software packages that are simple to create and install. They auto-update and are safe to run. And because they bundle their dependencies, they work on all major Linux systems without modification.{% endblock %}">
+    <meta name="copydoc" content="{% block meta_copydoc %}https://drive.google.com/drive/folders/1q9ia7iDuWQlCaJ7nfKVeQ5G-FPyJbP-H{% endblock %}">
```

Comment:
maybe put the `{% block meta_copydoc %}` around the whole line? Or does the extension rely on having an empty `copydoc` meta tag?

Reply:
Ignore me, that's the default to the folder.

Reply:
Yep, the idea is to have a folder as default link in case some page doesn't define a specific copy doc.